kubernetes/pro/application/ui-service.yaml

```diff
@@ -3,7 +3,7 @@ kind: Service
 metadata:
   labels:
     app: ui
-  name: ui
+  name: standard-ui
```

Comment:
Why this change?

Reply:
ui change was a mistake, fixed with 36a4f4f, security context fixed with 8688832